gulpfile.js

```diff
@@ -757,16 +757,15 @@ function printNobuildHelp() {
 }
 
 /**
- * Prints a helpful message that lets the developer know how to switch configs.
+ * Prints a helpful message that lets the developer know how to enable canary.
  * @param {string} command Command being run.
  */
-function printConfigHelp(command) {
+function printCanaryHelp(command) {
   if (!process.env.TRAVIS) {
-    log(green('Building the runtime for local testing with the'),
-        cyan((argv.config === 'canary') ? 'canary' : 'prod'),
-        green('AMP config.'));
-    log(green('⤷ Use'), cyan('--config={canary|prod}'), green('with your'),
-        cyan(command), green('command to specify which config to apply.'));
+    log(green('Building the runtime for local testing in'),
+        cyan(argv.canary ? 'canary' : 'prod'), green('mode.'));
+    log(green('⤷ Use'), cyan('--canary'), green('with your'),
```

Comment:
nit: wrap this line with `if (argv.canary)` - no need to tell the users to use `--canary` when they already used `--canary` 😉

Reply:
Done, with `if (!argv.canary)`.